src/quo2/components/avatars/wallet_user_avatar.cljs

```diff
@@ -25,11 +25,11 @@
 (defn wallet-user-avatar
   "params, first name, last name, color, size
    and if it's dark or not!"
-  [{:keys [f-name l-name color size]
-    :or   {f-name "John"
-           l-name "Doe"
-           color  :red
-           size   :x-large}}]
+  [{:keys [f-name l-name customization-color size]
```

Comment:
Not your PR's fault, but these `f-name` and `l-name` are probably one of the worst abbreviations I've seen for first and last name.

Reply:
I know, and these components need to be refactored to foilow the `:size` value we recently agreed, so there's still room to improve the API